global.json

```diff
@@ -14,5 +14,14 @@
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.22122.3",
     "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.22122.3"
+  },
+  "workloads": {
+    "macos": {
+      "version": "12.1.301-preview.13.7",
+      "sources": [
+        "https://aka.ms/dotnet6/nuget/index.json",
+        "https://api.nuget.org/v3/index.json"
```

Comment:
We don't use references to nuget.org in our main NuGet.config to reduce package substitution attacks, do we need to do the same here?

Reply:
Hmm, good point. I'll look into the best approach here. I assume it will be using `dotnet-public` instead of nuget.org.

Reply:
Turns out the dotnet6 feed is sufficient. Changed it to look the same as in NuGet.config, and removed nuget.org feed.